src/util/simplify_expr_int.cpp

```diff
@@ -54,47 +54,47 @@ bool simplify_exprt::simplify_bswap(bswap_exprt &expr)
   return true;
 }
 
-static bool sum_expr(exprt &dest, const exprt &expr)
+//! produce a sum of two constant expressions of the same type
+//! \return 'true' iff this was successful
```

Comment:
That comment is actually wrong... It might be a good case to start using enums or `optionalt` instead.